src/MedTrackerAPI/Features/Devices/GetDevice.cs

```diff
@@ -0,0 +1,45 @@
+using MediatR;
+using MedTrackerAPI.Endpoints;
+using MedTrackerAPI.Infrastructure;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace MedTrackerAPI.Features.Devices;
+
+public static class GetDevice
+{
+    public record Response(int DeviceId, string Description, string Manufacturer, string Model, string PartNumber, string LotNumber);
+    
+    public sealed class Endpoint : IEndpoint
+    {
+        public void MapEndpoint(IEndpointRouteBuilder app)
+        {
+            app.MapGet("devices/{deviceId}", async (IMediator mediator, int deviceId) =>
+                {
+                    var query = new GetDeviceQuery { DeviceId = deviceId };
```

Comment:
I normally scrunch simple lines like this together w/ the next but that's my preference.